### DISCUSSION TITLE
Generating an "extension" tailwind CSS file

### DISCUSSION THREAD

**Discussion by stephenwf** [tailwindlabs/tailwindcss#13138] [Help]
A general question. 

If you generate a Tailwind CSS file, is it possible to create a second separate CSS file that contains only additional classes used by a second set of source files. 

Base application:
```
<div className="text-md bg-red-500">...</div>
``` 

"Extension":
```
<div className="text-md bg-red-600">...</div>
```

Is it possible to produce:

- `style.css` - with the tailwind base + `.text-md` and `.bg-red-500`
- `extension.css` - with only `.bg-red-600` class

**Comment by wongjn** [comment]
You could consider creating a plugin for your build system or build system process that does this for you.
